specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```diff
@@ -0,0 +1,3315 @@
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete a test": {
+            "$ref": "./examples/DeleteLoadTest.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Test"
+        ],
+        "summary": "Get load test details by test name",
+        "operationId": "Test_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/TestIdPathParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The requested test.",
+            "schema": {
+              "$ref": "#/definitions/Test"
+            }
+          },
+          "default": {
+            "description": "Load Testing service error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a test by an unique name": {
+            "$ref": "./examples/GetLoadTest.json"
+          }
+        }
+      }
+    },
+    "/tests": {
+      "get": {
+        "tags": [
+          "Test"
+        ],
+        "summary": "Get all load tests by the fully qualified resource Id e.g subscriptions/{subId}/resourceGroups/{rg}/providers/Microsoft.LoadTestService/loadtests/{resName}.",
+        "operationId": "Test_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "orderBy",
+            "description": "Sort on one of the field - lastModifiedDateTime, displayName, createdBy in (field asc/desc) format. eg: displayName asc.",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "search",
+            "description": "Filter search based on searchable fields - testId, createdBy.",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "lastUpdatedStartTime",
+            "description": "Start DateTime(ISO 8601 literal format) of the last updated time range to filter tests.",
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "in": "query",
+            "name": "lastUpdatedEndTime",
+            "description": "End DateTime(ISO 8601 literal format) of the last updated time range to filter tests.",
+            "type": "string",
+            "format": "date-time"
+          },
```

Comment:
With the exception of these two parameters, the service uses "lastModified" rather than "lastUpdated" in parameter/property names. For consistency I recommend you change these to "lastModified".

Reply:
used as lastModified